master/getting-started/openstack/installation/redhat.md

```diff
@@ -3,19 +3,6 @@ title: Red Hat Enterprise Linux packaged install
 canonical_url: 'https://docs.projectcalico.org/v2.6/getting-started/openstack/installation/redhat'
 ---
 
-For this version of {{site.prodname}}, with OpenStack on RHEL or CentOS, we recommend
-using OpenStack Liberty or later.
-
-> **Note**: On RHEL/CentOS 7.3, with Mitaka or earlier, there is a Nova
-> [bug](https://bugs.launchpad.net/nova/+bug/1649527) that breaks {{site.prodname}}
-> operation. You can avoid this bug by:
->
-> - Using Newton or later (recommended), or
->
-> - Manually [patching](https://review.openstack.org/#/c/425637/) your Nova
->   install on each compute node.
-{: .alert .alert-info}
```

Comment:
I think we still need this - has it moved to somewhere else?

(Sorry for not catching this earlier!)

Reply:
OK I can add it back.... it is just that we don't have the official support for these older versions, but I understand. No worries, let me add it back.